### PR TITLE
Added a manpage

### PR DESCRIPTION
**Pull request by Linux4Life531**
## Standards checklist:

- [Y] The PR title is descriptive.
- [Y] The code compiles
- [Have not tested] The code passes rustfmt
- [Have not tested] The code passes clippy
- [N] The code passes tests
- [Y] *Optional:* I have tested the code myself
    - [N] I also tested that Topgrade skips the step where needed

I have added the 2 files (One must be ran on download which I do not know how to set it to run automatically and the other is the actual script). I have touched nothing else and it should run as normal. I have tested the normal topgrade command and it does work. Thank you.

By the way, this is my first pull request so if I do or have done anything wrong please let me know.

### CODE REVIEW

**Comment by Linux4Life531** [comment]
This was to fix issue #301.

**Comment by r-darwish** [comment]
Please post the uncompressed manual and remove your script. Package maintainers should include this man page instead of using a script

**Comment by Linux4Life531** [comment]
I have unzipped the man page file and added the zipping process into the MustBeRanOnDownloadIfOnLinux.sh file. If you know how to automatically make it run this on download, please let me know as it would be a great help for users of this project. Thank you.

**Comment by Linux4Life531** [comment]
> Please post the uncompressed manual and remove your script. Package maintainers should include this man page instead of using a script

When you install it through the package manager, will it automatically set up the man page for you when it installs? This would mean you wouldn't need my other file, the bash script, that sets up everything.

**Comment by Linux4Life531** [comment]
Sorry for tall of the new commits, I accidentally merged some of what I am working on into the version I have put this pr in for. I have deleted all of the changes. Sorry.

File: topgrade.8 [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 .TH man 8 "22 Febuary 2020" "1.0" " Topgrade Manpage"
 .SH NAME
-topgrade \- upgrade everything on your computer
+topgrade - upgrade everything on your computer
 .SH SYNOPSIS
 topgrade
 .SH DESCRIPTION
 Keeping your system up to date mostly involves invoking more than a single package manager. This usually results in big shell one-liners saved in your shell history. Topgrade tries to solve this problem by detecting which tools you use and run their appropriate package managers.
 .SH OPTIONS
 The topgrade does not take any options.
```
Comment (r-darwish) [inline topgrade.8]: There are arguments. Please run `topgrade --help` to see them

**Comment by Linux4Life531** [comment]
Please could you post a list of whatever you want me to include. I do not know any of the arguments for topgrade which is why I am asking. You can also give me anything to include in other sections. Thank you.

File: topgrade.8 (hunk 1 above)
Comment (Linux4Life531) [reply topgrade.8]: Okay.

Comment (Linux4Life531) [reply topgrade.8]: Okay.

Comment (Linux4Life531) [reply topgrade.8]: I think I have added them in. Thanks

**Comment by Linux4Life531** [comment]
Is there anything that anyone else would like me to add?

**Comment by Linux4Life531** [comment]
I have deleted it.

**Comment by Linux4Life531** [comment]
Is there anything else that you want me to do? Anyone can ask.

**Comment by Linux4Life531** [comment]
> The man page seems okay. Please delete the bash script as we won't need it. Package maintainers will include this man page in their packages.

I have deleted the script, thanks for letting me know.

**Comment by Linux4Life531** [comment]
Is there anything anyone else wants me to add/do? Feel free to ask.

**Comment by r-darwish** [comment]
Thank you @Linux4Life531

**Comment by Linux4Life531** [comment]
It has been a pleasure.

**Pull request merged**